### ISSUE TITLE
[BUG] `dlopen failed: cannot locate symbol "PyFloat_Type"`

### ISSUE THREAD

**Issue by Kry9toN** [dashingsoft/pyarmor#1615]
### I am unable to obfuscate a python script using pyarmor
Logs
```
INFO     Python 3.11.6
INFO     Pyarmor 8.4.4 (trial), 000000, non-profits
INFO     Platform linux.aarch64
INFO     search inputs ...
INFO     find package at .
INFO     find 1 top resources
ERROR    dlopen failed: cannot locate symbol "PyFloat_Type" referenced by "/data/data/com.termux/files/usr/lib/python3.11/site-packages/pyarmor/cli/core/android/aarch64/pytransform3.so"...
```

### What steps will reproduce the bug?
`pyarmor gen -O obf -r --exclude './dist' --exclude './build' . --private -e $(date -d "+30 days" "+%Y-%m-%d")`

### Running on device
Android (termux)

**Comment by jondy** [comment]
Could you check the symbol of Python3.11 in termnux?
```
objdump -T /path/to/libpython3.11.so | grep PyFloat
```

**Comment by jondy** [comment]
Maybe it need `--enable-shared` to build Python for termux

**Comment by Kry9toN** [comment]
but on running the script runtime is error

```
Traceback (most recent call last):
  File "/data/data/com.termux/files/home/source/out/main.py", line 2, in <module>
    from pyarmor_runtime_000000 import _pyarmor_                                 File "/data/data/com.termux/files/home/source/out/pyarmor_runtime_000000/_init.py", line 2, in <module>                                                            from .pyarmor_runtime import __pyarmor_
ImportError: dlopen failed: library "libpthread.so.0" not found: needed by /data/data/com.termux/files/home/source/out/pyarmor_runtime_000000/pyarmor_runtime.so in namespace (default)
```

ldd pyarmor_runtime.so
        libpthread.so.0 => not found
        libdl.so.2 => not found
        libutil.so.1 => not found
        libm.so.6 => not found
        libc.so.6 => not found

**Comment by Kry9toN** [comment]
i realize if i make for linux.aarch64 so i generate again with these command

`pyarmor gen -O out -r --exclude './dist' --exclude './build' . --private -e $(date -d "+30 days" "+%Y-%m-%d") --platform android.aarch64`

but is still error while run the program
```
Traceback (most recent call last):
  File "/data/data/com.termux/files/home/klik/out/main.py", line 2, in <module>      from pyarmor_runtime_000000 import _pyarmor_
  File "/data/data/com.termux/files/home/klik/out/pyarmor_runtime_000000/_init_.py", line 2, in <module>
    from .pyarmor_runtime import _pyarmor_                                     ImportError: dlopen failed: cannot locate symbol "PyBytes_AsStringAndSize" referenced by "/data/data/com.termux/files/home/klik/out/pyarmor_runtime_000000/pyarmor_runtime.so"...
```

**Comment by jondy** [comment]
Also `patchelf --add-needed libpython3.11.so.1.0 /path/to/pyarmor_runtime.so`

**Comment by Kry9toN** [comment]
yes also that command and work perfect

**Comment by itsmenewbie03** [comment]
> Also `patchelf --add-needed libpython3.11.so.1.0 /path/to/pyarmor_runtime.so`

This fixed my issue except I'm using python 3.12
